pkg/handlers/dev/generate.go

```diff
@@ -0,0 +1,116 @@
+package dev
+
+import (
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/aserto-dev/aserto/pkg/cc"
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/pkg/errors"
+)
+
+type GenerateFromOpenAPI struct {
+	URL  string `arg:"" required:"" help:"URL of the openapi.yaml"`
+	Name string `arg:"" optional:"" help:"The name for the policy"`
+}
+
+const packageTemplate = `package %s
+
+default allowed = false`
+
+func parseURI(uri string) []string {
+	result := []string{}
+	parts := strings.Split(uri, "/")
+	for _, part := range parts[1:] {
+		if strings.Contains(part, "{") {
+			clean := strings.Replace(strings.Replace(part, "{", "", -1), "}", "", -1)
+			result = append(result, "__"+clean)
+		} else {
+			result = append(result, part)
+		}
+	}
+	return result
+}
+
+func generatePackageName(root, verb, uri string) string {
+	parts := []string{root, verb}
+	parts = append(parts, parseURI(uri)...)
+	return strings.Join(parts, ".")
+}
+
+func (cmd *GenerateFromOpenAPI) Run(c *cc.CommonCtx) error {
+
+	specURL, err := url.Parse(cmd.URL)
+	if err != nil {
+		log.Fatal("Failed to parse spec URL", err)
```

Comment:
`log.Fatal` prints a message and exits the program immediately. It should only be used in `main` or perhaps functions directly called by it, not deep in sub packages.
Return an error and let the top-level code deal with it, just like the other errors in this function.